backend/corpora/common/entities/collection.py

```diff
@@ -66,7 +72,7 @@ def list_collections_in_time_range(cls, *args, **kwargs):
 
     @classmethod
     def list_attributes_in_time_range(
-        cls, to_date: int = None, from_date: int = None, filters: list = None, list_attributes: list = None
+            cls, to_date: int = None, from_date: int = None, filters: list = None, list_attributes: list = None
```

Comment:
???

Reply:
I think this is a pycharm formatting change?